### PR TITLE
Updates montage example to reflect ToDoMVC project changes

### PR DESCRIPTION
**Pull request by marchant**
PR to update MontageJS implementation to latest project npm dependencies as well as upgrade project to latest montage version

### CODE REVIEW

File: examples/montage/assets/app.css [hunk 1]
```diff
@@ -1,3 +1 @@
-#montage-todomvc .visible {
-	display: block;
-}
+#montage-todomvc .visible{display:block}
```
Comment (sindresorhus) [inline examples/montage/assets/app.css]: nope

File: examples/montage/index.html [hunk 2]
```diff
@@ -1,26 +1,18 @@
...
-			}
-		</script>
-	</body>
-</html>
+        <script src="packages/todomvc-common@13305a5/base.js"></script>
+        <script src="index.html.bundle-0.js" data-montage-location="packages/montage@23c6465/" data-promise-location="packages/q@833279d/" data-montage-hash="23c6465" data-promise-hash="833279d" data-application-hash="ad457b1"></script>
+        <script type="text/montage-serialization">{"owner":{"prototype":"montage/ui/loader.reel","properties":{"element":{"#":"todo-container"}}}}</script>
+    
+
+</body></html>
```
Comment (sindresorhus) [inline examples/montage/index.html]: Please follow our template and code style: https://github.com/tastejs/todomvc-app-template/blob/06a7873ca21a449bcb16036fdb84e6b06afed840/index.html

There's a lot of things wrong with these changes.

File: examples/montage/ui/main.reel/main.css [hunk 3]
```diff
@@ -1,5 +1 @@
-#main,
-#footer,
-#clear-completed-container {
-	display: none;
-}
+#main,#footer,#clear-completed-container{display:none}
```
Comment (sindresorhus) [inline examples/montage/ui/main.reel/main.css]: Formatted please.

**Comment by marchant** [comment]
@sindresorhus the committed version is the version ready for production, that's why the code is made more compact, it's not the source form, and it wasn't before either. I can make it looks prettier if you want, just explaining.

File: examples/montage/index.html (hunk 2 above)
Comment (marchant) [inline examples/montage/index.html]: What would you like to see here? Montage is based on components and the index.html won't look quite like https://github.com/tastejs/todomvc-app-template/blob/06a7873ca21a449bcb16036fdb84e6b06afed840/index.html in source form, but it does when the app runs

Comment (sindresorhus) [inline examples/montage/index.html]: I didn't realize, but still applies to the source, like https://github.com/marchant/todomvc/blob/master/examples/montage/ui/main.reel/main.html

Comment (marchant) [inline examples/montage/index.html]: Ok, but what about it, indentation of tags? Not closing tags?

**Comment by arthurvr** [comment]
For the reference, this fixes #1222.

**Comment by arthurvr** [comment]
> that's why the code is made more compact, it's not the source form, and it wasn't before either.

@marchant Please take a look at https://github.com/tastejs/todomvc/issues/378. We don't want minified code in the repo ;)

**Comment by samccone** [comment]
when the test PR lands we can stack a style linter on top :) thus preventing this kind of review since CI will take care of it for us :dancer:

File: examples/montage/index.html (hunk 2 above)
Comment (arthurvr) [inline examples/montage/index.html]: > indentation of tags? Not closing tags?

Both. Also that file uses the wrong indentation. We use tabs.

File: examples/montage/packages/todomvc-common@13305a5/readme.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+## Install
+
+```
+$ npm install --save todomvc-common
+```
+
+
+## License
+
+MIT © [TasteJS](http://tastejs.com)
```
Comment (arthurvr) [inline examples/montage/packages/todomvc-common@13305a5/readme.md]: We only want the vendor files which we need to be committed. [We ignore the rest.](https://github.com/marchant/todomvc/blob/master/examples/mithril/.gitignore)

Comment (marchant) [inline examples/montage/packages/todomvc-common@13305a5/readme.md]: ok, so I shall remove every file that ends up not being used when the app runs, correct?

Comment (samccone) [inline examples/montage/packages/todomvc-common@13305a5/readme.md]: that is correct :dancer:

Comment (arthurvr) [inline examples/montage/packages/todomvc-common@13305a5/readme.md]: Also, Include them in the `gitignore` as seen in the link I posted.

Comment (marchant) [inline examples/montage/packages/todomvc-common@13305a5/readme.md]: OK, will do

Comment (arthurvr) [inline examples/montage/packages/todomvc-common@13305a5/readme.md]: Thanks!

Comment (marchant) [inline examples/montage/packages/todomvc-common@13305a5/readme.md]: @arthurvr @samccone I'm going to commit code that is not minified and addresses the format concerns, but that won't make it more they way you actually code Montage, which has been  available in the standalone repo there: https://github.com/montagejs/todo-mvc

**Comment by marchant** [comment]
Updated PR per recommendations, take a look and let me know

File: examples/montage/assets/app.css [hunk 5]
```diff
@@ -1,3 +1,3 @@
 #montage-todomvc .visible {
-	display: block;
+    display: block;
```
Comment (arthurvr) [inline examples/montage/assets/app.css]: We use tabs, this is 4 spaces.

Comment (marchant) [inline examples/montage/assets/app.css]: Missed some then! On it

Comment (arthurvr) [inline examples/montage/assets/app.css]: Thanks.

Comment (marchant) [inline examples/montage/assets/app.css]: done!

File: examples/montage/index.html [hunk 6]
```diff
@@ -2,25 +2,28 @@
 <html lang="en" data-framework="montage" id="montage-todomvc">
 	<head>
 		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (arthurvr) [inline examples/montage/index.html]: Why did you added this? https://github.com/tastejs/todomvc/commit/9429daa2d4ce051c8e95a34d4244b0e100e6c803

File: examples/montage/index.html [hunk 7]
```diff
@@ -2,25 +2,28 @@
...
-		<script type="text/montage-serialization">
-			{
-				"owner": {
-					"prototype": "montage/ui/loader.reel",
-					"properties": {
-						"element": {"#": "todo-container"}
+	<script src="packages/todomvc-common@8ebafa5/base.js"></script>
+	<script src="index.html.bundle-0.js" data-montage-location="packages/montage@604e6eb/" data-promise-location="packages/q@4636076/" data-montage-hash="604e6eb" data-promise-hash="4636076" data-application-hash="184f06d"></script>
+	<script type="text/montage-serialization">
+				{
```
Comment (arthurvr) [inline examples/montage/index.html]: weird indentation

File: examples/montage/ui/main.reel/main.css [hunk 8]
```diff
@@ -1,5 +1,5 @@
 #main,
 #footer,
 #clear-completed-container {
-	display: none;
+    display: none;
```
Comment (arthurvr) [inline examples/montage/ui/main.reel/main.css]: indentation is still wrong.

File: examples/montage/readme.md [hunk 9]
```diff
@@ -1,49 +1,69 @@
...
 Get help from other Montage users:
 
-* [IRC](http://webchat.freenode.net/?channels=montage)
-* [MontageJS on Google Groups](https://groups.google.com/forum/?hl=en&fromgroups#!forum/montagejs)
-* [MontageJS on Twitter](http://twitter.com/montagejs)
-* [MontageJS on Google +](https://plus.google.com/116915300739108010954)
+* [Forum](http://forum.montagestudio.com/)
+* [Montage Studio on Twitter](http://twitter.com/montagejs)
+* [Montage Studio on Facebook](https://www.facebook.com/montagestudio)
+* [Montage Studio on Google +](https://plus.google.com/+Montagestudio/about)
```
Comment (arthurvr) [inline examples/montage/readme.md]: Could you please extract unrelated changes to a separated commit?

**Comment by arthurvr** [comment]
Could you please update your PR description with a clear explanation of what you did?

**Comment by marchant** [comment]
was on vacation, sorry for the delay

**Comment by arthurvr** [comment]
@marchant ping? Could you re-look at my review comments? Seems like not everything was addressed yet.

**Comment by marchant** [comment]
Sorry Arthur, was on vacation, have more updates actually

> On Apr 24, 2015, at 12:54 PM, Arthur Verschaeve <notifications@github.com> wrote:
> 
> @marchant <https://github.com/marchant> ping? Could you re-look at my review comments? Seems like not everything was addressed yet.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/tastejs/todomvc/pull/1241#issuecomment-96045792>.
> 


 <http://montagestudio.com/>
montagestudio.com <http://montagestudio.com/>	 	
Benoit Marchant <mailto:benoit@montagestudio.com>
CEO & Founder
1290 Oakmead Parkway, Ste 111
Sunnyvale, CA 94085
(1) 408.621.4874 mobile <tel:+1-408-621-4874>
Profile <http://www.linkedin.com/in/benoitmarchant> |  @benoitmarchant <http://twitter.com/benoitmarchant>

**Comment by arthurvr** [comment]
@marchant There are still some outstanding things. When would you have time to fix those?

**Comment by marchant** [comment]
I’m working on an update with an upcoming release of MontageJS. So I can either address the current pending issues, and do another PR with the new version, or just wait  bit longer, what do you prefer?

> On May 9, 2015, at 11:35 PM, Arthur Verschaeve <notifications@github.com> wrote:
> 
> @marchant <https://github.com/marchant> There are still some outstanding things. When would you have time to fix those?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/tastejs/todomvc/pull/1241#issuecomment-100592604>.
> 


 <http://montagestudio.com/>
montagestudio.com <http://montagestudio.com/>	 	
Benoit Marchant <mailto:benoit@montagestudio.com>
CEO & Founder
1290 Oakmead Parkway, Ste 111
Sunnyvale, CA 94085
(1) 408.621.4874 mobile <tel:+1-408-621-4874>
Profile <http://www.linkedin.com/in/benoitmarchant> |  @benoitmarchant <http://twitter.com/benoitmarchant>

**Comment by arthurvr** [comment]
@marchant I haven't followed how big the update to the new version will be, but, I'd personally prefer to finish off this PR and fix those issues and later on come back to update to the latest version of montage.

File: examples/montage/ui/main.reel/main.css (hunk 8 above)
Comment (marchant) [inline examples/montage/ui/main.reel/main.css]: Indentation is 1 tab, what sure what's wrong?

Comment (arthurvr) [inline examples/montage/ui/main.reel/main.css]: nope, this is four spaces.

File: examples/montage/readme.md (hunk 9 above)
Comment (marchant) [inline examples/montage/readme.md]: This reflects changes around the project in general, it's related to update the montage example in general, not just about the new css

Comment (arthurvr) [inline examples/montage/readme.md]: > This reflects changes around the project in general, it's related to update the montage example in general, not just about the new css

In fact it's a little sad that all changes just went together in one big commit and one big PR, though, okay.

File: examples/montage/.gitignore [hunk 10]
```diff
@@ -0,0 +1,6 @@
+node_modules/todomvc-app-css/*
+!node_modules/todomvc-app-css/index.css
+
+node_modules/todomvc-common/*
+!node_modules/todomvc-common/base.js
+!node_modules/todomvc-common/base.css
```
Comment (arthurvr) [inline examples/montage/.gitignore]: Those files don't exist no?

File: examples/montage/readme.md [hunk 11]
```diff
@@ -1,49 +1,69 @@
...
->
-> _[MontageJS - montagejs.org](http://montagejs.org)_
+> MontageJS is a client-side HTML5 framework for building rich single-page applications. It offers time-tested patterns and principles, a modular architecture, and a web designer friendly method to achieve a clean separation of concerns. MontageJS is the open source HTML5 framework with a dedicated, cloud-based, visual authoring environment. If you have a GitHub account, you're ready to play in Montage Studio. And just like GitHub, Montage Studio is free to use for open source repos. In the studio you will discover an easier, faster way to develop single page applications that exceed user expectations on all screens. You can serve your app from your Montage Studio's session on the different target devices you care about. As you add, remove configure components to your project, or event edit the CSS, Montage Studio apply your changes in real time to connected devices running you app, through the cloud. Instant responsive feedback!
```
Comment (arthurvr) [inline examples/montage/readme.md]: This seems like a pretty big text already. This should be the quick mission of the framework, extra explanation should go further in the readme.

File: examples/montage/readme.md [hunk 12]
```diff
@@ -1,49 +1,69 @@
...
------------- | -------------
+------------ | ------------- 
 assets | Contains global styles and the background image for the TodoMVC application.
 core | Contains the data model.
-index.html | Is the entry-point HTML document.
+index.html | Is the entry-point HTML document.  
 LICENSE.md | Contains copyright information.
-package.json | Describes your app and its dependencies.
-README.md | Describes the TodoMVC application.
+package.json | Describes your app and its dependencies. MontageJS uses CommonJS.
```
Comment (arthurvr) [inline examples/montage/readme.md]: There's no `package.json` file no?

File: examples/montage/readme.md [hunk 13]
```diff
@@ -1,49 +1,69 @@
...
+README.md | Describes the TodoMVC application.
```
Comment (arthurvr) [inline examples/montage/readme.md]: `readme.md`. Lowercase :)

**Comment by arthurvr** [comment]
ping @marchant ^

**Comment by arthurvr** [comment]
Hey @marchant - The montage app is totally broken currently, so if you or anyone else isn't able to fix this soon it will get removed as of https://github.com/tastejs/todomvc/commit/8a35c022d292a6aab3c2d121348a17ca5d90679d. Thanks.

**Comment by arthurvr** [comment]
No worries. @marchant messaged me to say he'll be able to finish this up soon. Thanks!

**Comment by arthurvr** [comment]
Remove `.idea` files.

File: examples/montage/index.html [hunk 14]
```diff
@@ -4,23 +4,25 @@
...
 		<title>Montage • TodoMVC</title>
 
-		<link rel="stylesheet" href="bower_components/todomvc-common/base.css">
+		<link rel="stylesheet" href="packages/todomvc-common@8ebafa5/base.css">
+		<link rel="stylesheet" href="packages/todomvc-app-css@f1b6ee7/index.css">
 		<link rel="stylesheet" href="assets/app.css">
 	</head>
 	<body>
-		<div data-montage-id="todo-container" id="todo-container"></div>
+	<div data-montage-id="todo-container" id="todo-container"></div>
```
Comment (arthurvr) [inline examples/montage/index.html]: Please indent inside of the body.

**Comment by samccone** [comment]
When testing this application I have found an odd bug.

Add three todos titled 1, 2, 3; Then clear all three of the todos, after that add two new todos, titled 1, 2, notice how the second todo is already in the edit state, showing an input instead of the normal display. 

This is a nasty lil bug :bug:

**Comment by samccone** [comment]
### Memory Leak Bug
--------

##### Steps  to reproduce

* Add one todo and remove it to prime the footer

* start timeline profiler
  * add three todos
  * delete three todos
  * (repeat above 2 more times)
* force GC
* stop timeline profiler


##### Results:

It appears that we are leaking DOM nodes
screen shot 2015-06-26 at 3 33 35 pm

Upon further investigation I have found the reason for the retention of the nodes to be here:

screen shot 2015-06-26 at 3 37 19 pm

**Comment by marchant** [comment]
Thanks Sam, much appreciated, I’ll look at it tomorrow

> On Jun 26, 2015, at 3:42 PM, Sam Saccone <notifications@github.com> wrote:
> 
> Memory Leak Bug
> 
> Steps to reproduce
> 
> Add one todo and remove it to prime the footer
> 
> start timeline profiler
> 
> add three todos
> delete three todos
> (repeat above 2 more times)
> force GC
> stop timeline profiler
> Results:
> 
> It appears that we are leaking DOM nodes
>  <https://cloud.githubusercontent.com/assets/883126/8388669/b9e884f8-1c19-11e5-863c-3f7d2612b7d5.png>
> Upon further investigation I have found the reason for the retention of the nodes to be here:
> 
>  <https://cloud.githubusercontent.com/assets/883126/8388680/efa21ff0-1c19-11e5-8de8-d07751e99a60.png>
> —
> Reply to this email directly or view it on GitHub <https://github.com/tastejs/todomvc/pull/1241#issuecomment-115910156>.
> 


 <http://montagestudio.com/>
montagestudio.com <http://montagestudio.com/>	 	
Benoit Marchant <mailto:benoit@montagestudio.com>
CEO & Founder
1290 Oakmead Parkway, Ste 111
Sunnyvale, CA 94085
(1) 408.621.4874 mobile <tel:+1-408-621-4874>
Profile <http://www.linkedin.com/in/benoitmarchant> |  @benoitmarchant <http://twitter.com/benoitmarchant>

**Comment by marchant** [comment]
Thanks for the testing Sam, I can’t reproduce on Chrome  43.0.2357.130 on Mac, in which browser did you see this happen?

> On Jun 26, 2015, at 3:40 PM, Sam Saccone <notifications@github.com> wrote:
> 
> When testing this application I have found an odd bug.
> 
> Add three todos titled 1, 2, 3; Then clear all three of the todos, after that add two new todos, titled 1, 2, notice how the second todo is already in the edit state, showing an input instead of the normal display.
> 
> This is a nasty lil bug 
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/tastejs/todomvc/pull/1241#issuecomment-115909838>.
> 


 <http://montagestudio.com/>
montagestudio.com <http://montagestudio.com/>	 	
Benoit Marchant <mailto:benoit@montagestudio.com>
CEO & Founder
1290 Oakmead Parkway, Ste 111
Sunnyvale, CA 94085
(1) 408.621.4874 mobile <tel:+1-408-621-4874>
Profile <http://www.linkedin.com/in/benoitmarchant> |  @benoitmarchant <http://twitter.com/benoitmarchant>

**Comment by samccone** [comment]
chrome canary in icognito mode with no profile.
If you are still unable to reproduce I will put together a video showing you the steps.

**Comment by arthurvr** [comment]
@marchant, was you able to reproduce this?

**Comment by marchant** [comment]
I was not, I did your steps:

Add three todos titled 1, 2, 3; Then clear all three of the todos, after that add two new todos, titled 1, 2

Nothing special...

in tried in  canary, in icognito mode, Version 45.0.2451.0 canary (64-bit)

Not sure what is different from your setup leading to that difference… 

Benoit

> On Jul 8, 2015, at 11:17 AM, Arthur Verschaeve <notifications@github.com> wrote:
> 
> @marchant <https://github.com/marchant>, was you able to reproduce this?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/tastejs/todomvc/pull/1241#issuecomment-119513341>.
> 


 <http://montagestudio.com/>
montagestudio.com <http://montagestudio.com/>	 	
Benoit Marchant <mailto:benoit@montagestudio.com>
CEO & Founder
1290 Oakmead Parkway, Ste 111
Sunnyvale, CA 94085
(1) 408.621.4874 mobile <tel:+1-408-621-4874>
Profile <http://www.linkedin.com/in/benoitmarchant> |  @benoitmarchant <http://twitter.com/benoitmarchant>

**Comment by samccone** [comment]
Looks like we are also missing styles currently
screen shot 2015-07-08 at 11 13 28 am

**Comment by samccone** [comment]
Hey @marchant 

I ran my automated test for leaks over this project and got these results:

```
initial snapshot: {"documents":3,"jsEventListeners":12,"jsHeapSizeUsed":9155128,"nodes":202}
end snapshot:     {"documents":3,"jsEventListeners":12,"jsHeapSizeUsed":12795248,"nodes":332}
````


You can run this locally via 

* `git remote add sam git@github.com:samccone/todomvc.git`
* `git rebase sam/sjs/leak`
* `cd tests`
* `npm i`
* `cd ..`
* `node tests/leaks.js`

Let me know how I can help!

**Comment by marchant** [comment]
Thanks Sam, what is your leak test measuring? We do some objects pooling in the framework, which we may not have tested in repeated tests.   I’m not saying there are no leaks, but I can see how that could create some red herrings.


> On Jul 8, 2015, at 5:23 PM, Sam Saccone <notifications@github.com> wrote:
> 
> Hey @marchant <https://github.com/marchant>
> I ran my automated test for leaks over this project and got these results:
> 
> initial snapshot: {"documents":3,"jsEventListeners":12,"jsHeapSizeUsed":9155128,"nodes":202}
> end snapshot:     {"documents":3,"jsEventListeners":12,"jsHeapSizeUsed":12795248,"nodes":332}
> You can run this locally via
> 
> git remote add sam git@github.com:samccone/todomvc.git
> git rebase sam/sjs/leak
> cd tests
> npm i
> cd ..
> node tests/leaks.js
> Let me know how I can help!
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/tastejs/todomvc/pull/1241#issuecomment-119623804>.
> 


 <http://montagestudio.com/>
montagestudio.com <http://montagestudio.com/>	 	
Benoit Marchant <mailto:benoit@montagestudio.com>
CEO & Founder
1290 Oakmead Parkway, Ste 111
Sunnyvale, CA 94085
(1) 408.621.4874 mobile <tel:+1-408-621-4874>
Profile <http://www.linkedin.com/in/benoitmarchant> |  @benoitmarchant <http://twitter.com/benoitmarchant>

**Comment by samccone** [comment]
Hi @marchant this measurement is directly from chrome timeline memory logging.

I expose it via this flag in chromedriver https://github.com/samccone/leaky/blob/master/lib/index.js#L36
And then extract it from the logs by this name key https://github.com/samccone/leaky/blob/master/lib/index.js#L14

Let me know if you have other questions!

**Comment by sindresorhus** [comment]
ping :)

**Comment by samccone** [comment]
oh hay @marchant :) just a friendly ping

**Comment by arthurvr** [comment]
This has been taking really long and it looks like now the PR is also going inactive. The app is behind in some other ways too, so I think it's getting time to remove the app (see [`contributing.md`](https://github.com/tastejs/todomvc/blob/master/contributing.md#submitting-a-new-app)). I'm going to close this PR due to inactivity, remove the app from the repo and close its issues. If a community member volunteers to maintain the app and brings it back in a PR with all issues fixed we'll consider re-adding it.

**Comment by marchant** [comment]
We’re on it, actually, in which way is the app behind? the PR is coming

> On Oct 1, 2015, at 10:19 PM, Arthur Verschaeve <notifications@github.com> wrote:
> 
> This has been taking really long and it looks like now the PR is also going inactive. The app is behind in some other ways too, so I think it's getting time to remove the app (see contributing.md <https://github.com/tastejs/todomvc/blob/master/contributing.md#submitting-a-new-app>). I'm going to close this PR due to inactivity, remove the app from the repo and close its issues. If a community member volunteers to maintain the app and brings it back in a PR with all issues fixed we'll consider re-adding it.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/tastejs/todomvc/pull/1241#issuecomment-144925120>.
> 


 <http://montagestudio.com/>
montagestudio.com <http://montagestudio.com/>	 	
Benoit Marchant <mailto:benoit@montagestudio.com>
CEO & Founder
1290 Oakmead Parkway, Ste 111
Sunnyvale, CA 94085
(1) 408.621.4874 mobile <tel:+1-408-621-4874>
Profile <http://www.linkedin.com/in/benoitmarchant> |  @benoitmarchant <http://twitter.com/benoitmarchant>

**Comment by samccone** [comment]
@marchant ping us when you push :)

**Comment by arthurvr** [comment]
Little sad to see that we don't get a response for longer than a month and 10 minutes after we drop it you're there. Anyway, before we re-add it I'd like to see *all* issues solved, not only https://github.com/tastejs/todomvc/issues/1222. This way I'm afraid we'll just get in exactly the same situation after a while.

That includes https://github.com/tastejs/todomvc/issues/1222, https://github.com/tastejs/todomvc/issues/1110, https://github.com/tastejs/todomvc/issues/1234 and maybe even https://github.com/tastejs/todomvc-app-css/issues/4.

**Comment by thibaultzanini** [comment]
We are sorry for our late reply. We have been pretty busy this last month. I'm going to try to fix all the issues and the conflicts for the next few days.

**Comment by samccone** [comment]
No worries @thibaultzanini thanks for the update

**Comment by thibaultzanini** [comment]
Hi! We have finally a new version that fixes all issues. Once again, sorry for the inconvenience. But, I'm having some odds conflicts with this branch when I try to rebase, probably because the montage example has been dropped temporarily. So, I was wondering, if I could close this PR and make a new on the top of master with the updated montage example?

**Comment by samccone** [comment]
@thibaultzanini if you can create a new branch locally that sits cleanly on master you can then go into this branch and run

git reset --hard my-clean-branch

and then push -f your branch up and you should be good to go :)

**Comment by thibaultzanini** [comment]
@samccone Oh sure If you don't mind about doing that. Thanks.

**Comment by arthurvr** [comment]
That's not really something we can do for you :) GitHub has excellent docs, FYI.

**Comment by samccone** [comment]
@thibaultzanini Looks like you are getting jscs errors because it is trying to lint a compiled file, can you add the offending file to our .jscs ignore?

**Comment by thibaultzanini** [comment]
I'm not sure to understand the problem here: `Error: Unable to find application root, did you start your local server?`

**Comment by samccone** [comment]
hey @thibaultzanini looks like our test suite is unable to find the root of your element

Here is the line that is failing?
https://github.com/tastejs/todomvc/blob/master/tests/page.js#L199-L220

Does this make sense?

**Comment by thibaultzanini** [comment]
@samccone I see, thanks.

**Comment by samccone** [comment]
Hey @thibaultzanini ok great this looks like some progress, so as you can see from CI there are a few failures with your implementation and also a missing vendored resource

`http://localhost:8000/examples/montage/packages/montage@af1b182/core/event/key-manager.load.js`


Let me know if you think anything is amiss with our suite, thanks again! :heart_eyes_cat:

**Comment by samccone** [comment]
Hey guys, I am going to close the PR again, just ping us when you push and think it is good to go. Thanks

**Comment by thibaultzanini** [comment]
@samccone. Okay, we are still fixing the tests.

**Comment by samccone** [comment]
no worries :)
